portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,16 +36,19 @@
   private static final String COMMENT_PROP = "comment";
   private static final String DISPLAY_PROP = "name";
   private static final String TIME_PROP = "timestamp";
+  private static final String MAX_COMMENT_PROP = "max-comments";
   private static final String DEFAULT_VAL = "";
 
    @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort(TIME_PROP, SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
+    int max = Integer.parseInt(request.getParameter(MAX_COMMENT_PROP));
     ImmutableList<Comment> comments = 
         Streams.stream(results.asIterable())
             .map(entity -> (makeComment(entity)))
```

Comment:
nit: you can use the .map(DataServlet::makeComment) syntax instead, which is preferable style-wise but logically equivalent to what you have here. This syntax can be used in general for .map() for single-argument methods. "DataServlet" refers to the class the method is defined in, while "makeComment" is the name of the method.

A bit more on this syntax as an FYI: it only works on single-argument methods where the argument type is equivalent to the type of the "object" reference in our stream (eg: "entity" has type Entity and makeComment will only take a single Entity, so we can use this syntax. If our makeComment took multiple arguments (say an Entity and a String), then we can't use this syntax)